api/room/schema_query.py

```diff
@@ -7,11 +7,14 @@
 from helpers.calendar.events import RoomSchedules
 from helpers.calendar.analytics import RoomStatistics  # noqa: E501
 from api.room.models import Room as RoomModel
+from helpers.auth.user_details import get_user_from_db
+from helpers.remote_rooms.remote_rooms_location import \
```

Comment:
Hello @verenceLola , Is there we can do away with this slash?

Reply:
You can do it by putting that import in a bracket and move it to a new line. But it's more or less the same. But the latter makes for a more readable code.

Reply:
@nakatuddesuzan I have implemented your feedback. @C3real-kill3r thank you for the hint.